src/function/memo.rs

```diff
@@ -340,6 +340,94 @@ where
     }
 }
 
+#[cfg(not(feature = "shuttle"))]
+mod persistence {
+    use crate::function::memo::Memo;
+    use crate::function::Configuration;
+    use crate::revision::AtomicRevision;
+    use crate::zalsa_local::QueryRevisions;
+
+    use serde::ser::SerializeStruct;
+    use serde::Deserialize;
+
+    impl<C> serde::Serialize for Memo<'_, C>
```

Comment:
Would it be possible to derive the serialize instead? I'm a bit worried that we'll forget to add fields here